core/src/main/scala/org/apache/spark/scheduler/EventLoggingListener.scala

```diff
@@ -283,10 +283,15 @@ private[spark] object EventLoggingListener extends Logging {
    *
    * @param logStream Raw output stream to the event log file.
    */
-  def initEventLog(logStream: OutputStream): Unit = {
+  def initEventLog(logStream: OutputStream, testing: Boolean,
```

Comment:
nit: style

See style for multi-line arguments in `ApplicationAttemptInfo`, for example. But see below.